samples/drivers/can/babbling/boards/s32z270dc2_rtu0_r52.conf

```diff
@@ -0,0 +1,4 @@
+# Copyright 2022 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SAMPLE_CAN_BABBLING_FD_MODE=y
```

Comment:
Why is this needed? This changes the default behaviour of the sample for this one board? Instead of adding overlays and board config files to all the CAN samples, please enable CAN on the board level. This does not scale.

Reply:
yes, I enabled CAN on board level